### DISCUSSION TITLE
exploitdb change URL

### DISCUSSION THREAD

**Discussion by MSQucenge** [Homebrew/discussions#3988] [Writing Formulae/Casks]
brew Formula is https://github.com/Homebrew/homebrew-core/blob/master/Formula/exploitdb.rb，but
`  desc "Database of public exploits and corresponding vulnerable software"
  homepage "https://www.exploit-db.com/"
  url "https://github.com/offensive-security/exploitdb.git",
      tag:      "2022-10-18",
      revision: "b4e29f1fae9fe4aaced5ae0d18ffce260a8165ff"
  version "2022-10-18"
  license "GPL-2.0-or-later"
  head "https://github.com/offensive-security/exploitdb.git", branch: "master"`
is old!
https://github.com/offensive-security/exploitdb.git is changed https://gitlab.com/exploit-database/exploitdb.git

brew 中的exploitdb 是不是已经过时了？ exploitdb 的地址已经更改到了gitlab

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by p-linnane** [comment]
https://github.com/Homebrew/homebrew-core/pull/116763
